lambda_handlers/request_job_handler.py

```diff
@@ -22,7 +22,7 @@ def _handle(self, event, context):
             vars = event['vars']
```

Comment:
Shouldn't use `vars` as a variable name because it is redefining a built-in Python function.